modules/SofaBoundaryCondition/ConstantForceField.inl

```diff
@@ -45,165 +48,396 @@ namespace forcefield
 
 template<class DataTypes>
 ConstantForceField<DataTypes>::ConstantForceField()
-    : d_indices(initData(&d_indices, "indices",
-                         "indices where the forces are applied"))
-
-    , d_indexFromEnd(initData(&d_indexFromEnd,(bool)false,"indexFromEnd",
-                              "Concerned DOFs indices are numbered from the end of the MState DOFs vector. (default=false)"))
-
-    , d_forces(initData(&d_forces, "forces",
-                        "applied forces at each point"))
-
-    , d_force(initData(&d_force, "force",
-                       "applied force to all points if forces attribute is not specified"))
-
-    , d_totalForce(initData(&d_totalForce, "totalForce",
-                            "total force for all points, will be distributed uniformly over points"))
-
-    , d_arrowSizeCoef(initData(&d_arrowSizeCoef,(SReal)0.0, "arrowSizeCoef",
-                               "Size of the drawn arrows (0->no arrows, sign->direction of drawing. (default=0)"))
-
-    , d_color(initData(&d_color, defaulttype::RGBAColor(0.2f,0.9f,0.3f,1.0f), "showColor",
-                       "Color for object display (default: [0.2,0.9,0.3,1.0])"))
+    : d_indices(initData(&d_indices, "indices", "indices where the forces are applied"))
+    , d_indexFromEnd(initData(&d_indexFromEnd,bool(false),"indexFromEnd", "Concerned DOFs indices are numbered from the end of the MState DOFs vector. (default=false)"))
+    , d_forces(initData(&d_forces, "forces", "applied forces at each point"))
+    , d_force(initData(&d_force, "force", "applied force to all points if forces attribute is not specified"))
+    , d_totalForce(initData(&d_totalForce, "totalForce", "total force for all points, will be distributed uniformly over points"))
+    , d_arrowSizeCoef(initData(&d_arrowSizeCoef,SReal(0.0), "arrowSizeCoef", "Size of the drawn arrows (0->no arrows, sign->direction of drawing. (default=0)"))
```

Comment:
rename to drawArrow (to be consistant with other "draw" activation feature in other component. ?